test/hotspot/jtreg/serviceability/sa/ClhsdbFindPC.java

```diff
@@ -182,6 +182,19 @@ private static void testFindPC(boolean withXcomp, boolean withCore) throws Excep
                                           methodAddr));
             runTest(withCore, cmds, expStrMap);
 
+            // Run "mem -v <addr>/30" on a Method*. The first line will look like:
+            //   Address 0x0000152e30403530: Method jdk/test/lib/apps/LingeredApp.steadyState(Ljava/lang/Object;)V@0x0000152e30403530
+            // Followed by lines displaying the memory contents, including interpretation
+            // of any contents that are addresses.
+            cmdStr = "mem -v " + methodAddr + "/30";
+            cmds = List.of(cmdStr);
+            expStrMap = new HashMap<>();
+            expStrMap.put(cmdStr, List.of("Method jdk/test/lib/apps/LingeredApp.steadyState",
+                                          methodAddr,
+                                          /* The following is from a field in the Method object. */
+                                          "In interpreter codelet: method entry point"));
+            runTest(withCore, cmds, expStrMap);
+
```

Comment:
Is there a test case for another variant of range: Address,Address ?

Reply:
No. We really never had any specific test cases for `examine`. ClhsdbFindPC does make use of it, but not for the addr,addr variant, so it was also omitted for `mem`. I did consider adding a test case for it and still can. I started down that path, but adding an incremental value to a hex String was getting a bit ugly, so I abandoned it.